### ISSUE TITLE
Build AppImages for the client

### ISSUE THREAD

**Issue by TheAssassin** [mumble-voip/mumble#4659]
Currently there's an AppImage build script for murmur. I'd love to see such a script for the client, too, especially for snapshots of the development versions, but also for the stable ones.

I love Mumble, and I'd like to just download new versions and check them out without having to go through the hassle of adding a PPA. Download, make executable, run -- that sounds like the right way. It can be done with the server (using those "static binaries"), so why not offer that for the client, too?

**Comment by Krzmbrzl** [comment]
The AppImage we are currently building is broken. Thus we would appreciate if someone (with AppImage experience) could look into this as we currently don't have the capacities to fix it.

**Comment by TheAssassin** [comment]
@Krzmbrzl link please.

**Comment by Krzmbrzl** [comment]
The AppImage is always built as part of our CI job on Azure Pipelines. It can be found as a published "artifact" on the respective CI build overview.
The latest build can be found at https://dev.azure.com/Mumble-VoIP/Mumble/_build/results?buildId=3200&view=results

**Comment by trymeouteh** [comment]
Would like to see an AppImage for Mumble.

Windows users have the ability to download the installers for Mumble client install the client offline and backup the EXE installers. Mumble makes for a good communication app and service over mesh networks and having the ability to backup the client and server is essential if the internet goes down to deploy Mumble.

**Comment by codecnotsupported** [comment]
> The AppImage is always built as part of our CI job on Azure Pipelines. It can be found as a published "artifact" on the respective CI build overview. The latest build can be found at https://dev.azure.com/Mumble-VoIP/Mumble/_build/results?buildId=3200&view=results

I don't see any artifacts.

**Comment by Krzmbrzl** [comment]
Yeah, we no longer build the AppImage as it was broken and nobody has had the tine yet to look into fixing it.
